_includes/provider-ocp-mgt-token.md

```diff
@@ -1,18 +1,21 @@
 Run the following to obtain the token needed to add a Red Hat OpenShift provider:
 
-1.  Obtain the `management` service account token name:
+1. Create the service account token
+    ```bash
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+     name: $service_account_name-secret
+     namespace: $project_name
+     annotations:
+       kubernetes.io/service-account.name: $service_account_name
+    type: kubernetes.io/service-account-token
+    EOF
+    ```
+2. Describe the secret to retrieve the full token value:
 
-        # oc describe sa -n $project_name $service_account_name
-        ...
-        Tokens:  management-admin-token-0f3fh
-                 management-admin-token-q7a87
-
-2.  Select and describe one of the tokens to retrieve the full token
-    output, replacing `management-admin-token-0f3fh` with the name of
-    your token:
-
-        # oc describe secret -n $project_name management-admin-token-0f3fh
-        ...
-        Data
-        ====
-        token:  eyJhbGciOiJSUzI1NiI...
+   ```bash
+   oc get secret -o template $service_account_name-secret -n $project_name --template={{.data.token}
```

Comment:
think there's a typo in the template. Also, I think it's clearer if the template lives next to the -o template, so:


```suggestion
   oc get secret -n $project_name -o template --template={{.data.token}} $service_account_name-secret
```

Reply:
Oh yeah missed last character while copy&pasting, fixed